helm/qontract-reconcile/templates/template.yaml

```diff
@@ -241,6 +241,21 @@ objects:
     name: qontract-reconcile-{{ $integration.name }}
   spec:
     schedule: "{{ $integration.cron }}"
+    {{- if $integration.concurrencyPolicy }}
+    concurrencyPolicy: {{ $integration.concurrencyPolicy }}
```

Comment:
to simplify I think you could do something like `{{ $integration.concurrencyPolicy | default "Allow" }}` and remove the if block completely

same for the other if/else blocks added after this one

Reply:
Awesome suggestion! I'm less familiar with Helm syntax. I'll go with the abbreviated conditional route as it hopefully will lend to easier reading. Thanks!

Reply:
@jfchevrette PTAL; TIA!

Reply:
new version updated